src/rrs/muddle.init

```diff
@@ -0,0 +1,23 @@
+
+;"RRS;MUDDLE4 INIT ASKS IF YOU WANT TO LOAD PPRINT, Y OR y FOR YES
+  EVERY THING ELSE IOS NO. IT ALSO MAKES SURE THAT ,NULL IS SET
+  CORRECTLY"                                                   
+                                                                
+<DEFINE SETNUL ()                                               
+ <COND (<NOT <GASSIGNED? NULL!->>                                 
+        <SETG NULL!-                                             
+        <INSERT <ATOM <ASCII 127>> <GET ROOT OBLIST>>>)>> 
+        ;"<INSERT <ATOM <ASCII 127>> <GET INITIAL OBLIST>>>)>>"
+                                                                
+                                                                                                                           
+<DEFINE STARTER4 ()                                
+ <RESET .INCHAN>                                   
+ <TERPRI>                                          
+ <PRINC "PPRINT? (Y OR N)">                        
+ <COND (<MEMQ <TYI> "Yy"> <FLOAD "RRS;PPRINT 2">)  
```

Comment:
Should this really reference version 2, or rather ">"?

Reply:
Thanks!  I made the updates you suggested.